receiver/memcachedreceiver/testdata/fake_stats.json

```diff
@@ -0,0 +1,103 @@
+{
+   "0.0.0.0:55003":{
+       "Stats":{
+          "accepting_conns":"1",
+          "active_slabs":"11",
+          "auth_cmds":"11",
+          "auth_errors":"11",
+          "bytes":"11",
+          "bytes_read":"7",
+          "bytes_written":"11",
+          "cas_badval":"11",
+          "cas_hits":"11",
+          "cas_misses":"11",
+          "cmd_flush":"11",
+          "cmd_get":"11",
+          "cmd_meta":"11",
+          "cmd_set":"11",
+          "cmd_touch":"11",
+          "conn_yields":"11",
```

Comment:
All being the same value doesn't really show that we're mapping them correctly in the scraper, does it?

Reply:
Updated so all values should be different.